python/cuml/multiclass/multiclass.py

```diff
@@ -35,6 +36,28 @@ class MulticlassClassifier(Base, ClassifierMixin):
     copies back and forth the device and the host have some overhead. For more
     details see issue https://github.com/rapidsai/cuml/issues/2876.
 
+    Examples
+    --------
+    .. code-block:: python
+
+        from cuml.linear_model import LogisticRegression
+        from cuml.multiclass import MulticlassClassifier
+        from cuml.datasets.classification import make_classification
+
+        X, y = make_classification(n_samples=10, n_features=6, n_informative=4,
+                                   n_classes=3, random_state=137)
+
+        cls = MulticlassClassifier(LogisticRegression(), strategy='ovo')
+        cls.fit(X,y)
+
+        print("Predictions", cls.predict(X))
+
+    Output:
+
+    .. code-block:: none
+
+        Predictions [1 1 1 0 0 2 2 2 0 1]
+
```

Comment:
Example looks great but can you convert this to the doctest format? Here is an example where I show the difference: https://github.com/rapidsai/cuml/issues/2415#issuecomment-676595598